src/primary_backup/main.rs

```diff
@@ -107,12 +121,10 @@ mod tests {
 
     #[tokio::test(flavor = "multi_thread")]
     async fn test_only_primary_server() {
+        fs::remove_dir_all(".primary1").unwrap_or_default();
```

Comment:
this shouldn't be necessary, there's a remove_dir_all already in the int function